homeassistant/components/onewire/sensor.py

```diff
@@ -246,9 +277,13 @@ def get_entities(onewirehub: OneWireHub, config):
             device_type = device["type"]
             device_id = os.path.split(os.path.split(device["path"])[0])[1]
             dev_type = "std"
+            device_path = device["path"]
```

Comment:
Since you are creating a variable, can you reuse it throughout the function (eg. line 306 and 313)?

Reply:
Good point. Changed lines 306 and 313 to use the variable instead. That was all i could find.